### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by Sgdickerson93** [community/community#134761] [New to GitHub]
Hello,

learning more about github, so is publishing a release merging changes into the targeted branch??

**Comment by Sgdickerson93** [comment]
or does it give users a chance to review the code changes and apply them and/or create their own pull requests to merge the changes?

**Comment by romhenri** [comment]
Releases are used to release a built and/or ready-to-use version.
Since traditionally, the repo stores the source code.

This merge, branch logic is about source code. Any question, i m here.
